front/app/containers/Admin/reporting/components/ReportBuilder/Widgets/ChartWidgets/MethodsUsedWidget/MethodsUsedCard.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+
+import { Box } from '@citizenlab/cl2-component-library';
+
+import { useMethodsUsed } from 'api/graph_data_units';
+import { ParticipationMethod } from 'api/phases/types';
+
+import Statistic from 'components/admin/Graphs/Statistic';
+
+import { MessageDescriptor, useIntl } from 'utils/cl-intl';
+
+import MissingData from '../../_shared/MissingData';
+import { getDaysInRange } from '../utils';
+
+import messages from './messages';
+import { Props } from './typings';
+
+const PARTICIPATION_METHODS: ParticipationMethod[] = [
+  'information',
+  'ideation',
+  'native_survey',
+  'voting',
+  'volunteering',
+  'poll',
+  'survey',
+  'document_annotation',
+];
+
+const LABELS: Record<ParticipationMethod, MessageDescriptor> = {
+  information: messages.information,
+  ideation: messages.ideation,
+  native_survey: messages.nativeSurvey,
+  voting: messages.voting,
+  volunteering: messages.volunteering,
+  poll: messages.poll,
+  survey: messages.survey,
+  document_annotation: messages.document_annotation,
```

Comment:
What is this method? I'm not familiar with it.

Reply:
Konveio!